jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -318,6 +352,7 @@ YUI.add('user-profile-entity', function() {
           name: name,
           owner: entity.owner
         });
+        destroyButton = this._generateDestroyButton(entity);
```

Comment:
Since we already have the isModel check in place for other stuff, I took advantage of that to do one less function call for each entity that are not models.